apps/web/app/api/checkNftProof/route.ts

```diff
@@ -0,0 +1,31 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { Address } from 'viem';
+import { getKv } from 'apps/web/src/utils/datastores/kv';
+import { logger } from 'apps/web/src/utils/logger';
+
+type RequestBody = {
+  address: Address;
+};
+
+export async function POST(request: NextRequest) {
+  const { address } = (await request.json()) as RequestBody;
```

Comment:
nit: If the request body is "null" (for example), this will throw an error. I'd put this in the try/catch, personally